adam/learner/object_recognizer.py

```diff
@@ -199,9 +200,33 @@ def match_objects(
         else:
             object_names_to_patterns = self._object_names_to_static_patterns
 
+        # We special case handling the ground perception
+        # Because we don't want to remove it from the graph, we just want to use it's
```

Comment:
```suggestion
        # Because we don't want to remove it from the graph, we just want to use its
```

Reply:
Also it might be good to describe in the comment the situation which motivated us to do this.